egs/vaevc/vcc2018v1/local/download.sh

```diff
@@ -0,0 +1,68 @@
+#! /bin/bash
+#
+# download.sh
+# Copyright (C) 2020 Wen-Chin HUANG
+#
+# Distributed under terms of the MIT license.
+#
+
+db_root=$1
+
+# shellcheck disable=SC1091
+. utils/parse_options.sh || exit 1
+set -eu # stop when error occured and undefined vars are used
+
+wavdir=wav
+textdir=text
+[ ! -e ${db_root}/${wavdir} ] && mkdir -p ${db_root}/${wavdir}
+[ ! -e ${db_root}/${textdir} ] && mkdir -p ${db_root}/${textdir}
+
+if [ ! -e ${db_root}/.done ]; then
+    cd ${db_root}
+    
+    # download and decompress 
+    wget https://datashare.is.ed.ac.uk/bitstream/handle/10283/3061/vcc2018_database_training.zip
+    wget https://datashare.is.ed.ac.uk/bitstream/handle/10283/3061/vcc2018_database_evaluation.zip
+    wget https://datashare.is.ed.ac.uk/bitstream/handle/10283/3061/vcc2018_database_reference.zip
+    wget https://datashare.is.ed.ac.uk/bitstream/handle/10283/3061/vcc2018_database_evaluation_transcriptions.tar.gz
+    unzip -q vcc2018_database_training.zip
+    unzip -q vcc2018_database_evaluation.zip
+    unzip -q vcc2018_database_reference.zip
+    tar xf vcc2018_database_evaluation_transcriptions.tar.gz
+
+    # copy the wav
+    for spk in vcc2018_training/VCC2*; do
+        mkdir -p ${wavdir}/$(basename ${spk})
+        cp ${spk}/*.wav ${wavdir}/$(basename ${spk})
+    done
+    for spk in vcc2018_evaluation/VCC2*; do
+        cp ${spk}/*.wav ${wavdir}/$(basename ${spk})
+    done
+    for spk in vcc2018_reference/VCC2*; do
+        cp ${spk}/*.wav ${wavdir}/$(basename ${spk})
+    done
+    
+    # rename the folders
+    cd ${wavdir}
+    rename "s/VCC2//g" *
```

Comment:
I do not have ```rename``` command. Is it possible to change it by substitute?

Reply:
Can I know what platform you are using? I didn't know that `rename` is platform-specific...

Reply:
Ubuntu 18.04.4 LTS as well as Mac OSX.

Reply:
Okay. I changed to use `mv`.